Amplitude/Amplitude.m

```diff
@@ -739,11 +742,27 @@ - (void)turnOffSessionLaterExecute
 
 - (void)setUserProperties:(NSDictionary*) userProperties
 {
+    [self setUserProperties:userProperties replace:YES];
```

Comment:
It's a change in behavior but I think the default for replace should be NO, as it's more expected by people